### PR TITLE
add news about first release

### PR DESCRIPTION
**Pull request by lilyjazz**
## Why are these changes needed?

add news about first release

## Checks

- [x] I've included any doc changes needed for https://github.com/pingcap/LinguFlow/.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

**Review by lucklove — DISMISSED** [review]

**Review by shhdgit — APPROVED** [review]
lgtm. cheers!🎉

File: README.md [hunk 1]
```diff
@@ -33,11 +33,12 @@ In essence, LinguFlow's design and implementation method offer a structured and
 
 ### Localhost (docker)
 
-You can run LinguFlow on your local machine using [docker](https://docs.docker.com/get-docker/) compose. This setup is perfect for developing, testing LinguFlow applications, and diagnosing integration issues.
+You can run LinguFlow on your local machine using [docker compose](https://docs.docker.com/get-docker/). This setup is perfect for developing, testing LinguFlow applications, and diagnosing integration issues.
```
Comment (freedeaths) [inline README.md:36]: https://docs.docker.com/compose/install/     

The above link seems better.

Comment (lilyjazz) [reply README.md:36]: Thank you. I will change the link.

Comment (lilyjazz) [reply README.md]: PTAL Again~

**Review by freedeaths — APPROVED** [review]
LGTM

**Pull request merged**